extras/derbynet.conf

```diff
@@ -29,5 +29,32 @@
 # BARCODE_SCANNER_DEV
 
 # For derby-timer.sh script:
-# TIMER_USER=Timer
-# TIMER_PASSWORD=
+# TIMER_ARGS passes arguments to derby-timer.jar. Normal arugments can be used. See usage below.
+# TIMER_ARGS="AddArgumentsHere"
```

Comment:
I think a better format would be to show active or commented-out examples of setting various settings.  E.g.,
```
TIMER_ARGS=
TIMER_ARGS="$TIMER_ARGS TIMER_USER=Timer"
TIMER_ARGS="$TIMER_ARGS TIMER_PASSWORD= "
# TIMER_ARGS="$TIMER_ARGS -n /dev/ttyUSB0"
```
...